src/python/module/z5py/file.py

```diff
@@ -18,8 +18,8 @@ class File(Base):
     zarr_exts = {'.zarr', '.zr'}
     n5_exts = {'.n5'}
 
-    @staticmethod
-    def infer_format(path):
+    @classmethod
+    def infer_format(cls, path):
```

Comment:
I noticed that there was a hard-coded reference to File from the inside the method, and given that subclassing was getting involved I thought it would be safer to use an explicitly passed-in class reference rather than a hard-coded reference to File.

Reply:
Ok, I guess it shouldn't make much difference, the change is fine with me.